packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -845,24 +1027,23 @@ class IOSDeviceLogReader extends DeviceLogReader {
   /// is true.
   final List<String> _streamFlutterMessages = <String>[];
 
-  /// When using both `idevicesyslog` and `ios-deploy`, exclude logs with the
-  /// "flutter:" prefix if they have already been added to the stream. This is
-  /// to prevent duplicates from being printed.
-  ///
-  /// If a message does not have the prefix, exclude it if the message's
-  /// source is `idevicesyslog`. This is done because `ios-deploy` and
-  /// `idevicesyslog` often have different prefixes on non-flutter messages
-  /// and are often not critical for CI tests.
+  /// When using multiple logging sources, exclude logs with a `flutter:` prefix
```

Comment:
i think the original doc is clearer that it specifies what exactly "multiple" logging sources are

Reply:
Well it's no longer just the two, we now potentially use the Dart VM as well, but I'll try and update it to be more clear